creator-node/src/apiHelpers.js

```diff
@@ -191,3 +191,73 @@ module.exports.errorResponseNotFound = (message) => {
 module.exports.errorResponseSocketTimeout = (socketTimeout) => {
   return errorResponse(500, `${socketTimeout} socket timeout exceeded for request`)
 }
+
+/**
+ * Define custom api error subclasses to be thrown in components and handled in route controllers
+ */
+
+class ErrorBadRequest extends Error {}
```

Comment:
jw is this required to raise errors in the component model? bc i see that we just have a direct call to the existing error in healthCheckController https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/components/healthCheck/healthCheckController.js#L33

Reply:
resolved offline, this is required